blocks/library/image/block.js

```diff
@@ -252,11 +252,19 @@ class ImageBlock extends Component {
 									bottomLeft: 'wp-block-image__resize-handler-bottom-left',
 								} }
 								enable={ { top: false, right: true, bottom: false, left: false, topRight: true, bottomRight: true, bottomLeft: true, topLeft: true } }
+								onResizeStart={ () => {
+									if ( ! selectionDisable ) {
+										toggleSelection();
```

Comment:
Maybe the usage should be to specify the value we want in toggling to avoid the need for the condition? `toggleSelection( false );`. Could it be clearer if we're consistent with what we're allowing here? A bit confusing to work with a disabled prop to toggle it as enabled; wondering if it should be `selectionDisabled`, `toggleSelectionDisabled`, or the inverse: `selectionEnabled`, `toggleSelection`.